src/test/AgentSimController.test.ts

```diff
@@ -1,3 +1,4 @@
+import "regenerator-runtime/runtime.js";
```

Comment:
Did you have to import this because you got a "regeneratorRuntime is not defined" error? If so, I added a Babel plugin that fixes that in the PR I just merged, so these imports may not be necessary anymore?

Reply:
Yup.